custom.css

```diff
@@ -192,6 +203,60 @@ nav a:active {
   }
 }
 
+/* Popup Styling start */
+.model-body {
+  position: relative;
+  width: 320px;
+  height: 90%;
```

Comment:
- [OPTIONAL] Good job implementing all the changes suggested by the previous reviewers. :+1: :+1: , I noticed that while implementing the last changes, you made adjustments to the width of your pop-up. Please you need to fix this as the pop-up covers over 90% of the screen when it is active according to the Figma design.
- You used a fixed width, which is not advisable as your pop-up need to scale proportionally across different screens.
- Your pop-up: -
![Selection_007](https://user-images.githubusercontent.com/107632780/232200815-9bb47fc9-feea-497c-9fef-352bce136b66.png)
- You can fix this by changing the with from `width: 320px` to `width: 90%`. :smile: :small_airplane: